src/xo-app/backup/edit/index.js

```diff
@@ -1,33 +1,31 @@
-import React, { Component } from 'react'
-import { getJob } from 'xo'
+import Component from 'base-component'
+import React from 'react'
+import { getJob, getSchedule } from 'xo'
 
 import New from '../new'
 
 export default class Edit extends Component {
-  constructor (props) {
-    super(props)
-    this.state = {}
-  }
-
   componentWillMount () {
     const { id } = this.props.routeParams
 
     if (id == null) {
       return
     }
 
-    getJob(id).then(job => {
-      this.setState({ job })
+    getSchedule(id).then(schedule => {
+      getJob(schedule.job).then(job => {
+        this.setState({ job, schedule })
+      })
```

Comment:
One day I would love to have configurable subscriptions which would allow for instance:

- subscribing to a single schedule
- subscribing to VM stats